### ISSUE TITLE
build.sh fails: No such file ‘netcdf-c-4.9.0.tar.gz’.

### ISSUE THREAD

**Issue by hakonhagland** [StrawberryPerl/build-extlibs#34]
When I run `build.sh 5034 __` I get the following output:
```
ContainerAdministrator@4afdf4701a3b MINGW64 /z/extlib
# ./build.sh 5034 __
###### [18:00:42] BUILD STARTED param1='5034' param2='__'
###### gcc: Target: x86_64-w64-mingw32
###### is64bit: 1
###### out: /z/extlib/_5034__
#### Unpacking sources
### termcap-1.3.1
2023-02-16 18:00:45 URL:https://ftp.gnu.org/gnu/termcap/termcap-1.3.1.tar.gz [439416/439416] -> "termcap-1.3.1.tar.gz" [1]
### readline-8.2
2023-02-16 18:00:49 URL:https://ftp.gnu.org/gnu/readline/readline-8.2.tar.gz [3043952/3043952] -> "readline-8.2.tar.gz" [1]
GONNA Apply: /z/extlib/patches/readline-8.2/01-mingw-patch.patch : /z/extlib/_5034__/readline-8.2.patch.log

[...]

GONNA Apply: /z/extlib/patches/hdf-4.2.15/001-mingw.diff : /z/extlib/_5034__/hdf-4.2.15.patch.log
GONNA Apply: /z/extlib/patches/hdf-4.2.15/003-17.diff : /z/extlib/_5034__/hdf-4.2.15.patch.log
GONNA Apply: /z/extlib/patches/hdf-4.2.15/111-jpeg-INT32-collision.diff : /z/extlib/_5034__/hdf-4.2.15.patch.log
### hdf5-1.12.2
2023-02-16 18:08:48 URL:https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-1.12/hdf5-1.12.2/src/hdf5-1.12.2.tar.gz [13964914/1396491
4] -> "hdf5-1.12.2.tar.gz" [1]
GONNA Apply: /z/extlib/patches/hdf5-1.12.2/hdf5-fix-find-szip.patch : /z/extlib/_5034__/hdf5-1.12.2.patch.log
GONNA Apply: /z/extlib/patches/hdf5-1.12.2/hdf5-proper-library-names-mingw.patch : /z/extlib/_5034__/hdf5-1.12.2.patch.log
### netcdf-c-4.9.0
No such file ‘netcdf-c-4.9.0.tar.gz’.

FATAL: source tarball for 'netcdf-c-4.9.0' not found
```
I believe the first error message:

> No such file ‘netcdf-c-4.9.0.tar.gz’.

comes from line 227 in `build.sh`:

https://github.com/StrawberryPerl/build-extlibs/blob/04ca82c0c34464882a3515fbe0f690fcd3227e45/build.sh#L227

since if I run the wget manually like this:

```
# wget -nv -nc --no-check-certificate ftp://ftp.unidata.ucar.edu/pub/netcdf/netcdf-c-4.9.0.tar.gz
No such file ‘netcdf-c-4.9.0.tar.gz’.
```
I get the same error. It seems like the ftp site does not exist any more and they have moved the files here instead:

https://github.com/Unidata/netcdf-c/releases

**Comment by hakonhagland** [comment]
Added a possible fix here: See #35

**Comment by hakonhagland** [comment]
Closing this since #35 was merged.
